Access Parameter store config locally via Dotenv (#4745)
guardian/dotcom-rendering: dotcom-rendering/docs/architecture/028-dotenv.md

# `.env` File

> Any values in the `.env` file [shouldn't be required to run the application locally](../values/README.md#env-shouldnt-be-required) - to keep is at approachable as possible.

Reviewer: ```suggestion
Any values in the `.env` file [shouldn't be required to run the application locally](../values/README.md#env-shouldnt-be-required) - to keep it as approachable as possible.
```

👩‍🏫
